### DISCUSSION TITLE
Question on disabling decreased window size with escape key and jump search

### DISCUSSION THREAD

**Discussion by YuumiPie** [TT-ReBORN/Georgia-ReBORN#173] [Q&A] [answered]
As I want foobar to be always fullscreen, is there any way to prevent georgia-reborn from entering windowed mode when clicking the escape key? I could not find any options of changing shortcuts in the .json config file. I also could not find any way to disable the jump search feature in json, but there seems to be some ways you can remove the search queries in player controls.

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @YuumiPie,

this was not configurable until now, I have added both options in the latest commit https://github.com/TT-ReBORN/Georgia-ReBORN/commit/c7228699631e3b4da8655d63c6003be6d50df5a0: 
- Added option to disable jump search via top menu `Options` > `Player controls` > `Jump search` > `Disable jump search`
- Added option to disable ESC key to exit fullscreen via top menu `Options` > `Player controls` > `Panel` > `Disable fullscreen ESC`

You can download it from the master and replace your files:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip

-TT

**Reply by YuumiPie** [reply]
Works perfect. Thanks!
